operators/appian-operator/v0.168.0/metadata/annotations.yaml

```diff
@@ -0,0 +1,11 @@
+annotations:
+  # Core bundle annotations.
+  operators.operatorframework.io.bundle.mediatype.v1: registry+v1
+  operators.operatorframework.io.bundle.manifests.v1: manifests/
+  operators.operatorframework.io.bundle.metadata.v1: metadata/
+  operators.operatorframework.io.bundle.package.v1: appian-operator
+  operators.operatorframework.io.bundle.channels.v1: alpha
+  operators.operatorframework.io.metrics.builder: operator-sdk-v1.38.0
+  operators.operatorframework.io.metrics.mediatype.v1: metrics+v1
+  operators.operatorframework.io.metrics.project_layout: go.kubebuilder.io/v4
+  com.redhat.openshift.versions: v4.12-v4.16
```

Comment:
I added this line manually this time. We have a ticket to add this on the appian-operator repo side: https://appian-eng.atlassian.net/browse/CN-33901